virttest/qemu_monitor.py

```diff
@@ -2783,6 +2853,38 @@ def block_dirty_bitmap_clear(self, node, name):
         """
         return self._operate_dirty_bitmap("clear", node, name)
 
+    def block_dirty_bitmap_merge(self, node, src_bitmaps, dst_bitmap):
+        """
+        Merge source bitmaps to target bitmap for given node
+
+        :param string node: block device node name
+        :parma list src_bitmaps: list of source bitmaps
+        :param string dst_bitmap: target bitmap name
+        :raise: MonitorNotSupportedCmdError if 'block-dirty-bitmap-mege' and
+                'x-block-dirty-bitmap-mege' commands not supported by QMP
+                monitor.  
+        """
+        if self._has_command("x-block-dirty-bitmap-merge"):
```

Comment:
I think it is not necessary to support the experimental feature, or at least we should introduce another method to distinguish them and make a wrapper for it.

Reply:
This feature needs to be tested on 7.7, so it is necessary to maintain compatibility.

Reply:
OK, then shall we have two different methods to introduce them? That because, with the developing of QEMU, the two commands may have the different argument list, it is inconvenient to manipulate them in the same one method.